x-pack/filebeat/filebeat.reference.yml

```diff
@@ -245,6 +245,18 @@ filebeat.modules:
     # Filebeat will choose the paths depending on your OS.
     #var.paths:
 
+#------------------------------ Kubernetes Module ------------------------------
+- module: kubernetes
+  # All logs
+  coredns: 
+    enabled: true
+  envoy:
+    enabled: true
```

Comment:
I think they should be two modules, one for coredns and another one for envoy.

Reply:
I like the suggestion. Let me see whether doing these two modules separately will allow them to work in kubernetes deployments. Some additional work might still be needed.